README.md

```diff
@@ -414,8 +441,9 @@ const addTodoProcess = createProcess([
 		calculateCountsCommand,
 		postTodoCommand,
 		calculateCountsCommand
-	],
-	{ callback: addTodoCallback });
+	], {
+	callback: addTodoCallback
```

Comment:
ha I think this is actually less clear (very subjective though)

Reply:
Reverted!